Formula/open-zwave.rb

```diff
@@ -1,132 +1,32 @@
 class OpenZwave < Formula
   desc "Library for selected Z-Wave PC controllers"
-  homepage "http://www.openzwave.com"
-  url "http://old.openzwave.com/downloads/openzwave-1.2.919.tar.gz"
-  sha256 "473229f3dd3d6b260e6584b17e5c5f2e09e61805f89763f486a9f7aa2b4181ba"
+  homepage "http://www.openzwave.net/"
+  url "https://github.com/OpenZWave/open-zwave/archive/v1.4.tar.gz"
```

Comment:
why the switch from the canonical URL?

Reply:
github openzwave repo has a tag only for 1.4 (without minor versions) so it was even hard to understand what is  `1.4.164` version  from old.openzwave.com.
Also the `old` prefix in `old.openzwave.com`  made me feel like it was some legacy or unused site.

Reply:
Please file an issue upstream and ask them what the correct, current stable URL is.